# RM

## Example of genomic structure

The RM system have been describe in a total of 5 subsystems.

Here is some example found in the RefSeq database:

<img src="./data/RM_Type_I.svg">

RM\_Type\_I subsystem in the genome of *Aeromonas veronii* (GCF\_014169835.1) is composed of 4 proteins: Type\_I\_MTases (WP\_182963881.1), Type\_I\_MTases (WP\_182963881.1), Type\_I\_S (WP\_182963883.1)and, Type\_I\_REases (WP\_182963884.1).

<img src="./data/RM_Type_II.svg">

RM\_Type\_II subsystem in the genome of *Mannheimia haemolytica* (GCF\_007965905.1) is composed of 2 proteins: Type\_II\_MTases (WP\_006248352.1)and, Type\_II\_REases (WP\_006253295.1).

<img src="./data/RM_Type_IIG.svg">

RM\_Type\_IIG subsystem in the genome of *Spirochaeta africana* (GCF\_000242595.2) is composed of 1 protein: Type\_IIG (WP\_014455422.1).

<img src="./data/RM_Type_III.svg">

RM\_Type\_III subsystem in the genome of *Pannonibacter phragmitetus* (GCF\_001484065.1) is composed of 2 proteins: Type\_III\_MTases (WP\_058898889.1)and, Type\_III\_REases (WP\_058898890.1).

<img src="./data/RM_Type_IV.svg">

RM\_Type\_IV subsystem in the genome of *Clostridioides difficile* (GCF\_018884605.1) is composed of 1 protein: Type\_IV\_REases (WP\_021364579.1).

## Distribution of the system among prokaryotes

The RM system is present in a total of 4699 different species.

Among the 22k complete genomes of RefSeq, this system is present in 19087 genomes (83.7 %).

<img src="./data/Distribution_RM.svg" width=800px>

*Proportion of genome encoding the RM system for the 14 phyla with more than 50 genomes in the RefSeq database.*  *Pie chart of the repartition of all the subsystems found in the RefSeq database.*

## Relevant abstracts

**Oliveira, P. H., Touchon, M. & Rocha, E. P. C. The interplay of restriction-modification systems with mobile genetic elements and their prokaryotic hosts. Nucleic Acids Res 42, 10618-10631 (2014).**
The roles of restriction-modification (R-M) systems in providing immunity against horizontal gene transfer (HGT) and in stabilizing mobile genetic elements (MGEs) have been much debated. However, few studies have precisely addressed the distribution of these systems in light of HGT, its mechanisms and its vectors. We analyzed the distribution of R-M systems in 2261 prokaryote genomes and found their frequency to be strongly dependent on the presence of MGEs, CRISPR-Cas systems, integrons and natural transformation. Yet R-M systems are rare in plasmids, in prophages and nearly absent from other phages. Their abundance depends on genome size for small genomes where it relates with HGT but saturates at two occurrences per genome. Chromosomal R-M systems might evolve under cycles of purifying and relaxed selection, where sequence conservation depends on the biochemical activity and complexity of the system and total gene loss is frequent. Surprisingly, analysis of 43 pan-genomes suggests that solitary R-M genes rarely arise from the degradation of R-M systems. Solitary genes are transferred by large MGEs, whereas complete systems are more frequently transferred autonomously or in small MGEs. Our results suggest means of testing the roles for R-M systems and their associations with MGEs.

