---
title: AVAST
---

## Description 
AVAST (antiviral ATPases/NTPases of the STAND superfamily) is a group of anti-phage defense systems, active against some dsDNA phages. 

AVAST systems are composed of NTPases of the STAND (signal transduction ATPases with numerous associated domains) superfamily (1).  STAND-NTPases typically contain a C-terminal helical sensor domain that activates the N-terminal effector domain upon target recognition (1).

In eukaryotes, STAND-NTPases are associated with programmed cell death, therefore Gao and colleagues hypothesized that AVAST might function through an Abortive infection mechanism.

## Example of genomic structure

The AVAST system have been describe in a total of 5 subsystems.

Here is some example found in the RefSeq database:


![AVAST_I](/AVAST_I.svg)

AVAST_I subsystem in the genome of *Vibrio sp.* (GCF\_905175355.1) is composed of 3 proteins: Avs1A (WP\_208445041.1), Avs1B (WP\_208445042.1)and, Avs1C (WP_108173272.1).


![AVAST_II](/AVAST_II.svg)

AVAST_II subsystem in the genome of *Escherichia coli* (GCF_018884505.1) is composed of 1 protein: Avs2A (WP_032199984.1).


![AVAST_III](/AVAST_III.svg)

AVAST_III subsystem in the genome of *Enterobacter cancerogenus* (GCF_002850575.1) is composed of 2 proteins: Avs3B (WP_199559884.1)and, Avs3A (WP_101737373.1).


![AVAST_IV](/AVAST_IV.svg)

AVAST_IV subsystem in the genome of *Escherichia coli* (GCF_016903595.1) is composed of 1 protein: Avs4A (WP_000240574.1).


![AVAST_V](/AVAST_V.svg)

AVAST_V subsystem in the genome of *Leclercia adecarboxylata* (GCF_006171285.1) is composed of 1 protein: Avs5A (WP_139565349.1).

## Distribution of the system among prokaryotes

The AVAST system is present in a total of 363 different species.

Among the 22k complete genomes of RefSeq, this system is present in 1046 genomes (4.6 %).

![Distribution_AVAST](/Distribution_AVAST.svg)

*Proportion of genome encoding the AVAST system for the 14 phyla with more than 50 genomes in the RefSeq database.*  *Pie chart of the repartition of all the subsystems found in the RefSeq database.*

## Experimental validation

AVAST systems were experimentally validated using:

Subsystem SIR2-STAND with a system from *Escherichia fergusonii's PICI (EfCIRHB19-C05)* in *Escherichia coli*  has an anti-phage effect against  T4, Lambda, HK97, HK544, HK578, T7 (Fillol-Salom et al., 2022)

Subsystem SIR2-STAND with a system from *Escherichia fergusonii's PICI (EfCIRHB19-C05)* in *Salmonella enterica *  has an anti-phage effect against  P22, BTP1, ES18, det7 (Fillol-Salom et al., 2022)

Subsystem SIR2-STAND with a system from *Escherichia fergusonii's PICI (EfCIRHB19-C05)* in *Klebsiella pneumoniae *  has an anti-phage effect against  Pokey (Fillol-Salom et al., 2022)

Subsystem Metallo beta-lactamase + protease + STAND (Type 1) with a system from *Erwinia piriflorinigrans* in *Escherichia coli*  has an anti-phage effect against  P1 (Gao et al., 2020)

Subsystem STAND (Type 2) with a system from *Escherichia coli* in *Escherichia coli*  has an anti-phage effect against  T4, P1 (Gao et al., 2020)

Subsystem DUF4297-STAND (Type 3) with a system from *Salmonella enterica* in *Escherichia coli*  has an anti-phage effect against  T2, T3, T7, PhiV-1 (Gao et al., 2020)

Subsystem Mrr-STAND (Type 4) with a system from *Escherichia coli* in *Escherichia coli*  has an anti-phage effect against  T3, T7, PhiV-1 (Gao et al., 2020)

Subsystem SIR2-STAND (Type 5) with a system from *Escherichia coli* in *Escherichia coli*  has an anti-phage effect against  T2 (Gao et al., 2020)

Subsystem SeAvs1 with a system from *Salmonella enterica* in *Escherichia coli*  has an anti-phage effect against  P1, ZL-19 (Gao et al., 2022)

Subsystem EcAcs1 with a system from *Escherichia coli* in *Escherichia coli*  has an anti-phage effect against  ZL-19 (Gao et al., 2022)

Subsystem EpAvs1 with a system from *Erwinia piriflorinigrans* in *Escherichia coli*  has an anti-phage effect against  P1, Lambda, , ZL-19 (Gao et al., 2022)

Subsystem SeAvs3 with a system from *Salmonella enterica* in *Escherichia coli*  has an anti-phage effect against  T7, PhiV-1, ZL-19 (Gao et al., 2022)

Subsystem KvAvs3 with a system from *Klebsiella variicola* in *Escherichia coli*  has an anti-phage effect against  P1, ZL-19 (Gao et al., 2022)

Subsystem EcAvs2 with a system from *Escherichia coli* in *Escherichia coli*  has an anti-phage effect against  T7, PhiV-1, P1, T4, T5, ZL-19 (Gao et al., 2022)

Subsystem Ec2Avs2 with a system from *Escherichia coli* in *Escherichia coli*  has an anti-phage effect against  P1 (Gao et al., 2022)

Subsystem EcAvs4 with a system from *Escherichia coli* in *Escherichia coli*  has an anti-phage effect against  T7, PhiV-1, ZL-19 (Gao et al., 2022)

Subsystem Ec2Avs4 with a system from *Escherichia coli* in *Escherichia coli*  has an anti-phage effect against  T7, PhiV-1, ZL-19 (Gao et al., 2022)

Subsystem KpAvs4 with a system from *Klebsiella pneumoniae* in *Escherichia coli*  has an anti-phage effect against  ZL-19 (Gao et al., 2022)

Subsystem CcAvs4 with a system from *Corallococcus coralloides* in *Escherichia coli*  has an anti-phage effect against  T7 (Gao et al., 2022)

## Relevant abstracts

**Gao, L. A. et al. Prokaryotic innate immunity through pattern recognition of conserved viral proteins. Science 377, eabm4096 (2022).**
Many organisms have evolved specialized immune pattern-recognition receptors, including nucleotide-binding oligomerization domain-like receptors (NLRs) of the STAND superfamily that are ubiquitous in plants, animals, and fungi. Although the roles of NLRs in eukaryotic immunity are well established, it is unknown whether prokaryotes use similar defense mechanisms. Here, we show that antiviral STAND (Avs) homologs in bacteria and archaea detect hallmark viral proteins, triggering Avs tetramerization and the activation of diverse N-terminal effector domains, including DNA endonucleases, to abrogate infection. Cryo-electron microscopy reveals that Avs sensor domains recognize conserved folds, active-site residues, and enzyme ligands, allowing a single Avs receptor to detect a wide variety of viruses. These findings extend the paradigm of pattern recognition of pathogen-specific proteins across all three domains of life.

**Gao, L. et al. Diverse enzymatic activities mediate antiviral immunity in prokaryotes. Science 369, 1077-1084 (2020).**
Bacteria and archaea are frequently attacked by viruses and other mobile genetic elements and rely on dedicated antiviral defense systems, such as restriction endonucleases and CRISPR, to survive. The enormous diversity of viruses suggests that more types of defense systems exist than are currently known. By systematic defense gene prediction and heterologous reconstitution, here we discover 29 widespread antiviral gene cassettes, collectively present in 32% of all sequenced bacterial and archaeal genomes, that mediate protection against specific bacteriophages. These systems incorporate enzymatic activities not previously implicated in antiviral defense, including RNA editing and retron satellite DNA synthesis. In addition, we computationally predict a diverse set of other putative defense genes that remain to be characterized. These results highlight an immense array of molecular functions that microbes use against viruses.

